Rakefile

```diff
@@ -23,10 +24,40 @@ task :setup do
   end
 end
 
+sources = Dir["lib/licensed/source/*.rb"].map { |f| File.basename(f, ".*") }
```

Comment:
I _think_ this may be relative to the `pwd`, not the file. To make things a bit more solid, it may be worthwhile to make the path absolute (e.g., `File.expand_path .... __dir__`)

Reply:
Testing locally, it looks `pwd` is the Gemfile dir when run with `bundler exec` but that ✨ being explicit